src/main.cpp

```diff
@@ -290,12 +290,6 @@ int main(int argc, char* argv[])
         qWarning() << "Can't init IPC, maybe we're in a jail? Continuing with reduced multi-client functionality.";
     }
 
-    // For the auto-updater
-    if (sodium_init() < 0) {
```

Comment:
I'm guessing you can remove the sodium include as well now? :x

Reply:
Yup, done.